osf/migrations/0135_merge_20180928_1919.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Look into: recreate file metadata migrations to get rid of the two merge migrations

Reply:
@erinspace, made a PR to address this: https://github.com/erinspace/osf.io/pull/31